Added bare-bones package.json and .gitignore

huck51 opened the pull request:
TODO:
-Add more dependencies
-Add scripts
-peer review

huck51 committed:
Added bare-bones package.json and .gitignore

TODO:
-Add more dependencies
-Add scripts
-peer review

huck51 commented:
Take a look at this and let me know if I need to make any changes. Also feel free to clone it down and make changes yourself.

agallant commented on package.json:
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "arc_hive",
+  "version": "1.0.0",
```
May want to set this to 0.0.0 for now - versions are subjective/convention, but often 1.0 means first stable complete release.

agallant commented on package.json:
```diff
@@ -0,0 +1,25 @@
+  "dependencies": {
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "mongoose": "^4.13.5",
```
Mongo may still make sense for some of your data persisting, but I also encourage checking out Airtable - https://www.npmjs.com/package/airtable

Sorry I've not given specifics yet, I'll try to get a copy of the Airtable base we're actually using and share it with you soon.

huck51 committed:
Updated package.json

ADDED:
-airtable

EDITED:
-changed version from 1.0.0 to 0.0.0

TODO:
-further peer review
-merge

loyslane reviewed (approved):
Looks good to me!
